sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java

```diff
@@ -49,7 +51,7 @@ private static void initializeCacheDirs(Context context, SentryOptions options)
     File cacheDir = new File(context.getCacheDir(), "sentry");
     cacheDir.mkdirs();
     options.setCacheDirPath(cacheDir.getAbsolutePath());
-    (new File(options.getOutboxPath())).mkdirs();
+    new File(options.getOutboxPath()).mkdirs();
```

Comment:
This will do I/O on the calling thread which likely will be the main thread since it's `init`.
Let's try to postpone this directory creation and try to do this in a background thread

Reply:
Yeah I see, we gotta fix this though I'll create an issue.